crates/ruma-client-api/src/sync/sync_events/v4.rs

```diff
@@ -232,30 +246,27 @@ pub struct SyncRequestList {
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
     pub sort: Vec<String>,
 
-    /// Required state for each room returned. An array of event type and state key tuples.
-    /// Note that elements of this array are NOT sticky so they must be specified in full when they
-    /// are changed. Sticky.
-    #[serde(default, skip_serializing_if = "Vec::is_empty")]
-    pub required_state: Vec<(TimelineEventType, String)>,
+    #[serde(flatten)]
+    /// The details to be included per room
+    pub room_details: RoomDetailsConfig,
 
-    /// The maximum number of timeline events to return per room. Sticky.
     #[serde(skip_serializing_if = "Option::is_none")]
-    pub timeline_limit: Option<UInt>,
+    /// If tombstoned rooms should be returned and if so, with what information attached
+    pub include_old_rooms: Option<IncludeOldRooms>,
 
     /// Filters to apply to the list before sorting. Sticky.
     #[serde(skip_serializing_if = "Option::is_none")]
     pub filters: Option<SyncRequestListFilters>,
 }
 
-/// The RoomSubscriptions of the SlidingSync Request
+/// Unified configuration for requesting room details.
```

Comment:
What is "unified" meant to communicate here?

Reply:
It means that we have generalized the `RoomConfiguration` from three similar copies in to one struct.

Reply:
So it's basically a historical note? I think I would find it less confusing we just dropped that one word.